src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -114,6 +114,28 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
           <DetailDisplay label="Expense Type" content={`${expenseType.code} - ${expenseType.name}`} />
         </Grid>
       </Grid>
+      <Grid container spacing={1} sx={{ marginTop: 2 }}>
+        <Grid item xs={4}>
+          <DetailDisplay label="Treasurer" content="" />
```

Comment:
Ok so because you're just using the label prop on these, you can go in and use part of this DetailDisplay component that's just the label since it's a custom component we made.